pom.xml

```diff
@@ -61,6 +61,7 @@
     <module>maven-failsafe-plugin</module>
     <module>maven-surefire-report-plugin</module>
     <module>surefire-its</module>
+    <module>surefire-extensions-spi</module>
```

Comment:
Why this is the last module?

Reply:
thx i will have a look

Reply:
The IDEA is adding new modules to the bottom. I will move it.